core/_neat.scss

```diff
@@ -0,0 +1,12 @@
+// Neat
+// ============================================================
+
+*, *:before, *:after {
```

Comment:
Begin pseudo elements with double colons: `::`<br>Each selector in a comma sequence should be on its own single line